client/src/components/Nav/NavLinks.jsx

```diff
@@ -77,7 +77,7 @@ export default function NavLinks({ clearSearch, isSearchEnabled }) {
                 <Menu.Item as="div">
                   <NavLink
                     className={cn(
-                      'flex w-full cursor-pointer items-center gap-3 px-3 py-3 text-sm text-white transition-colors duration-200 hover:bg-gray-700',
+                      'flex w-full cursor-pointer items-center gap-3 px-3 py-3 text-sm text-white transition-colors duration-200 hover:bg-gray-700 rounded-none',
```

Comment:
do we need the rounded-none class here? it wasn't there previously

Reply:
Yes, dannya made some changes to the css that kinda degraded the similarity between a few menu items, so I fixed them